### ISSUE TITLE
AVT 1 -  React MultiSelect w/initial selected item has DAP violations

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#4007]
## Environment
macOS Mojave version 10.14.5
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Carbon v10 - React
DAP IBM Accessibility July 2019 Ruleset

## Detailed Description
1. Go to the React MultiSelect w/initial selected item
2. Run DAP and the following issue is found (see screenshot):
<img width="1261" alt="Screen Shot 2019-09-16 at 3 36 27 PM" src="https://user-images.githubusercontent.com/21676914/64991861-4c67a400-d898-11e9-8953-003058112467.png">
Note: The X used to clear the select is missing a label. 
3. Click on the MultiSelect to expand the menu  and 7 DAP violations display (see screenshot):
<img width="1313" alt="Screen Shot 2019-09-16 at 3 47 38 PM" src="https://user-images.githubusercontent.com/21676914/64992426-7ec5d100-d899-11e9-91a3-03296e4c5753.png">
4. Click the X to clear the selection and expand the menu. 6 DAP violations display (see screenshot):
<img width="1133" alt="Screen Shot 2019-09-16 at 3 37 59 PM" src="https://user-images.githubusercontent.com/21676914/64992519-aa48bb80-d899-11e9-9297-b9c07461ef50.png">
